Symfony/CS/Tests/Fixer/AbstractFixerTestBase.php

```diff
@@ -60,8 +61,21 @@ protected function makeTest($expected, $input = null, \SplFileInfo $file = null,
             $this->assertSame($expected, $tokens->generateCode(), 'Code build on input code must match expected code.');
 
             Tokens::clearCache();
+            $expectedTokens = Tokens::fromCode($expected);
             $tokens->clearEmptyTokens();
-            $this->assertTokens(Tokens::fromCode($expected), $tokens);
+            $this->assertTokens($expectedTokens, $tokens);
+
+            $tokensReflection = new \ReflectionClass($expectedTokens);
+            $propertyReflection = $tokensReflection->getProperty('foundTokenKinds');
+            $propertyReflection->setAccessible(true);
+            $foundTokenKinds = array_keys($propertyReflection->getValue($expectedTokens));
```

Comment:
this looks wrong to me: you are checking that all registered token kinds are indeed registered as found. this won't detect mistakes forgetting to register a new token in it.

What you should do instead is looping over the actual tokens and asserting that they are registered (this could be done inside ``assertTokens`` to avoid looping twice)